es-serviced-addendum.yaml

```diff
@@ -3,18 +3,18 @@
 #
 # Disable dynamic scripting engine to address CVE-2014-3120 (CC-997)
 #
-script.disable_dynamic: true
-
-#
-# Disable groovy scripting engine to address CVE-2015-1427 (CC-997)
-#
-script.groovy.sandbox.enabled: false
+script.allowed_types: inline
```

Comment:
Why do we need inline scripts?  Don't want to revert security fixes.

Reply:
This was default suggestion after the previous setting was deprecated if there's no need for it then I can remove it at all